### PR TITLE
Removed forbidden import 'com.google.common.collect' in MANIFEST files where not needed

### PR DESCRIPTION
**Pull request by cweitkamp**
- Removed forbidden import 'com.google.common.collect' in MANIFEST files where not needed

wrt https://github.com/openhab/openhab-core/pull/407

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: addons/binding/org.openhab.binding.harmonyhub/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -24,7 +24,6 @@ Bundle-Vendor: openHAB
 Bundle-Version: 2.4.0.qualifier
 Import-Package: 
  com.google.common.base,
- com.google.common.collect,
```
Comment (wborn) [inline addons/binding/org.openhab.binding.harmonyhub/META-INF/MANIFEST.MF]: The harmony library also uses the collect package:

```
src/main/java/net/whistlingfish/harmony/protocol/MessageStartActivity.java:import com.google.common.collect.ImmutableMap;
src/main/java/net/whistlingfish/harmony/protocol/MessagePing.java:import com.google.common.collect.ImmutableMap;
src/main/java/net/whistlingfish/harmony/protocol/MessageAuth.java:import com.google.common.collect.ImmutableMap;
src/main/java/net/whistlingfish/harmony/protocol/MessageGetCurrentActivity.java:import com.google.common.collect.ImmutableMap;
src/main/java/net/whistlingfish/harmony/protocol/IrCommand.java:import com.google.common.collect.ImmutableMap;
src/main/java/net/whistlingfish/harmony/protocol/MessageHoldAction.java:import com.google.common.collect.ImmutableMap;
src/main/java/net/whistlingfish/harmony/config/HarmonyConfig.java:import static com.google.common.collect.Lists.newArrayList;
src/main/java/net/whistlingfish/harmony/config/HarmonyConfig.java:import static com.google.common.collect.Maps.newHashMap;
src/main/java/net/whistlingfish/harmony/config/Device.java:import static com.google.common.collect.Lists.newArrayList;
src/main/java/net/whistlingfish/harmony/config/Activity.java:import static com.google.common.collect.Lists.newArrayList;
src/main/java/net/whistlingfish/harmony/config/Activity.java:import static com.google.common.collect.Maps.newHashMap;
```

**Comment by cweitkamp** [comment]
@wborn I reverted all removals where I am not sure if the dependency is needed by thirdparty libs.

File: addons/binding/org.openhab.binding.orvibo/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -12,7 +12,6 @@ Bundle-SymbolicName: org.openhab.binding.orvibo;singleton:=true
 Bundle-Vendor: openHAB
 Bundle-Version: 2.4.0.qualifier
 Import-Package: 
- com.google.common.collect,
```
Comment (wborn) [inline addons/binding/org.openhab.binding.orvibo/META-INF/MANIFEST.MF]: It looks like [s20-sdk-0.0.6-SNAPSHOT.jar uses Guava](https://github.com/tavalin/orvibo-sdk/blob/0df7811510517307b5bcaa55d96247d9ee44e6db/pom.xml#-L68-L72).

Comment (wborn) [reply addons/binding/org.openhab.binding.orvibo/META-INF/MANIFEST.MF]: Okay, that was the POM for 1.1.2-SNAPSHOT, the [0.0.6-SNAPSHOT POM](https://github.com/tavalin/orvibo-sdk/blob/0.0.6/pom.xml) does not seem to use Guava. :upside_down_face:

**Pull request merged**